.github/workflows/dependencies.yml

```diff
@@ -33,4 +33,7 @@ jobs:
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b  # v3.0.2
 
       - name: Dependency Review ("Dependabot on PR")
-        uses: actions/dependency-review-action@a9c83d3af6b9031e20feba03b904645bb23d1dab  # v1.0.2
+        uses: actions/dependency-review-action@0efb1d1d84fc9633afcdaad14c485cbbc90ef46c  # v2.5.1
+        with:
+            allow-ghsas:
+              - GHSA-w596-4wvx-j9j6  # this refers a potential attack via Subversion; ignoring since we only use Git.
```

Comment:
[Reference](https://github.com/actions/dependency-review-action#allow-ghsas)

The documentation looks straightforward, but I'm having trouble getting this to work correctly.

Reply:
got it figured out now.